lib/utils/__tests__/isStandardSyntaxFunction.test.js

```diff
@@ -26,6 +26,30 @@ describe('isStandardSyntaxFunction', () => {
 			false,
 		);
 	});
+
+	it('CSS-in-JS interpolation', () => {
+		const functions = [];
+
+		valueParser('${({ size }) => (size === "small") ? "0.8em" : "1em"}').walk((valueNode) => {
```

Comment:
Because there is no CSS-in-JS test yet in the repository, skipping parsing CSS as with other tests in this file, and go directly to parsing the value.

Reply:
> go directly to parsing the value.

Good point. Furthermore, what about rewriting the `func` helper in this test file and the existing test cases as below?

```js
it('calc', () => {
	expect(isStandardSyntaxFunction(func('calc(a + b)'))).toBe(true);
});

// ...

it('CSS-in-JS syntax', () => {
	expect(isStandardSyntaxFunction(func('`calc(${token.radiusBase} + 2px)`'))).toBe(false);
});

// ...

function func(declValue) {
	const functions = [];

	valueParser(declValue).walk((valueNode) => {
		if (valueNode.type === 'function') {
			functions.push(valueNode);
		}
	});

	return functions[0];
}
```